### ISSUE TITLE
Loading `.vscode/LoadTerminal.json` relative to VS Code CWD not Project / Workspace directory 

### ISSUE THREAD

**Issue by tohagan** [ariassd/vscode-load-terminals#10]
In #6 I recorded details about this issue. 

After much trail and error trying to get `.vscode/LoadTerminals.json` to be read I found that it only works if VS Code is started up in the Project or Workspace folder despite having configured the Workspace setting `terminalLoader.config`  as below.  I think it needs to read this `directory` relative to the current Workspace root folder (if active) or Project root folder (if no Workspace) rather than the current directory.

```json
{
  "terminalLoader.config": {
    "directory": ".vscode"   // << won't work
    // "directory": "${workspaceFolder}/.vscode"  // still wont't work
  }
}
```

**Comment by ariassd** [comment]
This extension only supports workspace configuration.

Thank you.

**Comment by tohagan** [comment]
But it does not work **for workspaces** because it loads the file from the wrong path. 
I've only tested this for workspaces.

**Comment by leite08** [comment]
For whoever gets here, I was able to load the settings file with this setup:
- VSCode workspace Multi-root
- `LoadTerminal.json` located on `$workspace/.vscode`
- Workspace settings - file `project.code-workspace`:
  - `settings` > `terminalLoader.config` > `directory` = `<workspace-home>/.vscode`

I just set `<workspace-home>` to the folder name where the workspace is running. It then found the settings file.

Tried to use `${workspaceFolder}` and `${workspaceRoot}` (deprecated), but I think this extension doesn't recognize those.
